### ISSUE TITLE
(webview2-runtime) Outdated

### ISSUE THREAD

**Issue by michael-franck** [pauby/ChocoPackages#177]
### Checklist

- [X] I have checked the [moderation queue](https://community.chocolatey.org/packages?q=&moderatorQueue=true&moderationStatus=all-statuses&prerelease=false&sortOrder=package-download-count) and no new version has been submitted (only visible when logged in), or a version has been submitted but has failed the automated checks.

- [X] The new version is a stable release. Pre-release versions will not be added.
- [X] There are no existing [bug](https://github.com/pauby/ChocoPackages/issues?q=is%3Aissue+is%3Aopen+label%3Abug) or [outdated](https://github.com/pauby/ChocoPackages/issues?q=is%3Aissue+is%3Aopen+label%3Aoutdated) issues.
- [X] The new version has a Windows binary (EXE or MSI installer, Zip or other archive, etc).

### New Software Version

111.0.1661.62

**Comment by michael-franck** [comment]
Haha no problem.

Quick question: I'm guessing that as you are downloading the web installer instead of the full offline that it will always install the newest version of the software?

If so I can circumvent for now in my own builds.

**Comment by pauby** [comment]
> I'm guessing that as you are downloading the web installer instead of the full offline that it will always install the newest version of the software?

Yes. I don't like to do that, but there is a reason I'm doing it, and I can't remember what it was. _I have a sneaky feeling_ it was something about proper versioning. I could look at it again as part of this, if, that has changed.

What I'd love to do is bundle the offline installer into the package, but the license isn't available so can't confirm redistribution is allowed (it may also be > 200MB, I can't remember).

**Comment by michael-franck** [comment]
Offline installer is currently clocking in at 140MB. 

I guess your sneaky feeling could be right because if I remember they changed the installer without updating the version in one instance. Microsoft seems to like to do that.

I'm not that fluent in licensing terms so can't say for sure if you are allowed to distribute, but I think it is allowed: 
![image](https://user-images.githubusercontent.com/112862532/229761604-9c7c5b0f-fe23-43db-961f-acbf2f3a012a.png)


There is a document section detailing [distribution methods](https://learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/distribution)

**Comment by bedalus** [comment]
Getting the following issue, and not a one-off. Would it be safe to ignore the checksum?

```
 webview2-runtime v110.0.1587.69 [Approved]
 webview2-runtime package files install completed. Performing other installation steps.
 Downloading webview2-runtime
   from 'https://go.microsoft.com/fwlink/p/?[REDACTED]

 Download of MicrosoftEdgeWebview2Setup.exe (1.54 MB) completed.
 Error - hashes do not match. Actual value was 'ED544BB542723F729E9873F37F9AE440A57136D0C429D47F83FF494164EF42D9'.
 ERROR: Checksum for 'C:\Users\packer\AppData\Local\Temp\chocolatey\webview2-runtime\110.0.1587.69\MicrosoftEdgeWebview2Setup.exe' did not meet '53186ada18194b1b8a0adb7ccfff5c819548e6b814f97a3341587720d450922f' for checksum type 'sha256'. Consider passing the actual checksums through with --checksum --checksum64 once you validate the checksums are appropriate. A less secure option is to pass --ignore-checksums if necessary.
 The install of webview2-runtime was NOT successful.
 Error while running 'C:\ProgramData\chocolatey\lib\webview2-runtime\tools\chocolateyinstall.ps1'.
  See log for details.

 Chocolatey installed 20/21 packages. 1 packages failed.
  See the log for details (C:\ProgramData\chocolatey\logs\chocolatey.log).
  ```

**Comment by pauby** [comment]
Given it's outdated and it's using the latest updated installer, I'd suggest using `--ignore-checksum` would make sense.

**Comment by pauby** [comment]
This was a bit of a journey. The API has changed. Once I figured that out I managed to update the code. Then the build server being used in AppVeyor (Windows Server 2012 R2) has a problem with the [code for this package](https://stackoverflow.com/questions/36275618/why-is-invoke-webrequest-and-invoke-restmethod-failing-and-succeeding-at-the-sam#comment76632531_36368402). 

I updated the build image to Windows Server 2019. However, the AppVeyor build now takes 20 minutes rather than a few minutes. It seems to hang. It does eventually get through the build. I've tried to downgrade to 2016 and no longer updating the Chocolatey version on those servers but same issue. 

So, I'm unsure. But it builds.

The latest version,113.0.1774.57, is now submitted and approved.
